plugins/modules/system/sudoers.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+# Copyright: (c) 2019, Jon Ellis (@JonEllis0) <ellis.jp@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: sudoers
+short_description: Manage Linux sudoers files
+description:
+    - This module allows for the manipulation of Linux sudoers files
+author:
+    - "Jon Ellis (@JonEllis0) <ellis.jp@gmail.com>"
+options:
+    command:
```

Comment:
It is a list, after all ;-)
```suggestion
    commands:
```

Reply:
Agh, I toyed with this one before I got to this suggestion.
I originally left it as command as I've only ever used it singularly, but allowed lists as I read the spec allowed lists.
I've gone with your suggestion.